tests/helpers/model.py

```diff
@@ -128,9 +128,8 @@ def model_load_test(config: Union[DictConfig, ListConfig], datamodule: Lightning
 
     """
     loaded_model = get_model(config)  # get new model
-    # Assing the weight file to resume_from_checkpoint. When trainer is initialized, Trainer
-    # object will automatically load the weights.
-    config.trainer.resume_from_checkpoint = os.path.join(config.project.path, "weights/last.ckpt")
+
+    ckpt_path = str(os.path.join(config.project.path, "weights/last.ckpt"))
```

Comment:
I think we could stick to Path throughout the library
```suggestion
    ckpt_path = str(Path(config.project.path) / "weights" / "last.ckpt")
```